es/es-repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java

```diff
@@ -33,24 +33,22 @@
 import java.net.URISyntaxException;
 import java.util.Map;
 
-import static java.util.Collections.emptyMap;
-
 public class AzureBlobStore implements BlobStore {
 
+    private final static String CLIENT = "default";
+
     private final AzureStorageService service;
 
-    private final String clientName;
     private final String container;
     private final LocationMode locationMode;
 
     public AzureBlobStore(RepositoryMetaData metadata, AzureStorageService service) {
         this.container = Repository.CONTAINER_SETTING.get(metadata.settings());
-        this.clientName = Repository.CLIENT_NAME.get(metadata.settings());
         this.service = service;
         // locationMode is set per repository, not per client
         this.locationMode = Repository.LOCATION_MODE_SETTING.get(metadata.settings());
-        final Map<String, AzureStorageSettings> prevSettings = this.service.refreshAndClearCache(emptyMap());
-        final Map<String, AzureStorageSettings> newSettings = AzureStorageSettings.overrideLocationMode(prevSettings, this.locationMode);
+        final AzureStorageSettings prevSettings = this.service.refreshAndClearCache(null);
```

Comment:
Couldn't we also adapt the `AzureStorageService` so get rid of the multiple client support?